cli/src/main/java/com/devonfw/tools/ide/process/ProcessResult.java

```diff
@@ -67,19 +67,25 @@ default boolean isSuccessful() {
     return getExitCode() == SUCCESS;
   }
 
+  /**
+   * @return the first captured standard out. Will be {@code null} if not captured but redirected.
+   */
+  default String getSingleOutput() {
```

Comment:
added convenience method for this common use-case (get expected single line of output) and throw error if process failed.